bedrock/mozorg/templates/mozorg/about/includes/m24/news.html

```diff
@@ -20,11 +20,11 @@ <h2 class="m24-c-intro-title">{{ ftl('m24-about-read-the-latest') }}</h2>
             {
               'media': '(max-width: 767px)',
               'srcset': {
-                "img/mozorg/about/m24/news/projects-mobile-375.jpg": "375w",
-                "img/mozorg/about/m24/news/projects-mobile-720.jpg": "720w",
-                "img/mozorg/about/m24/news/projects-mobile-976.jpg": "976w",
-                "img/mozorg/about/m24/news/projects-mobile-1232.jpg": "1232w",
-                "img/mozorg/about/m24/news/projects-mobile-1450.jpg": "1450w",
+                "img/mozorg/about/m24/news/builders-mobile-375.jpg": "375w",
+                "img/mozorg/about/m24/news/builders-mobile-720.jpg": "720w",
+                "img/mozorg/about/m24/news/builders-mobile-976.jpg": "976w",
+                "img/mozorg/about/m24/news/builders-mobile-1232.jpg": "1232w",
+                "img/mozorg/about/m24/news/builders-mobile-1450.jpg": "1450w",
```

Comment:
These images should have the aspect ratio 2:1. So the largest should be 1450 x 725, for example.